### ISSUE TITLE
Missing PHP timezone for IcingaWeb2

### ISSUE THREAD

**Issue by MAccadia** [Icinga/icinga2-ansible#69]
IcingaWeb2 setup complains about php timezone not set (`http://[IP]/icingaweb2/setup > Modules`).

```
The PHP config `date.timezone' is not defined.
```

## Possible Solution
Something like this:
```yaml
  tasks:
    - name: Timezone in php.ini
      lineinfile:
        path: /etc/php/7.0/apache2/php.ini
        regexp: '^;?date.timezone ='
        line: "date.timezone = \"{{ lookup('file', '/etc/timezone') }}\""
     notify: restart apache2
```

## My Environment
```
ansible 2.4.3.0
python version = 2.7.12 (default, Dec  4 2017, 14:50:18) [GCC 5.4.0 20160609]
Ubuntu 16.04.4 LTS
```

**Comment by aflatto** [comment]
Hello 
This is being addressed by PR #66

**Comment by MAccadia** [comment]
Thanks. I haven't checked there. I'm not a github master yet ;-)

Using ansible_fact is indeed better than file lookup.
